README.md

```diff
@@ -1,50 +1,70 @@
-# Prism
+[![Prism is a set of packages for API mocking with OpenAPI v2 (formerly known as Swagger) and OpenAPI v3.](./examples/readme-header.svg)](https://stoplight.io/api-mocking?utm_source=github&utm_medium=prism&utm_campaign=readme)
 
 [![CircleCI](https://img.shields.io/circleci/build/github/stoplightio/prism/master)](https://circleci.com/gh/stoplightio/prism)
```

Comment:
(Also referring to your previous comment) I think a better idea would be to use deferred links, such as

[Link][1]

```markdown
[Link][1]

[1]: http://google.it
```

so that we clean up this section from the long strings.

Reply:
Done